### PR TITLE
Fix: unhealthy container

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

**Review by vgorkavenko — CHANGES_REQUESTED** [review]
It shouldn't be like this. `Pulse` is needed for using by docker healthcheck because it is like a simple approach to ensure that everything is fine with application, so it shouldn't be requested inside a code (otherwise we obtain a recursive dependence).

Let's fix `Dockerfile` and nothing else:
From:
```
CMD curl -f http://localhost:$HEALTHCHECK_SERVER_PORT/healthcheck || exit 1
```
To:
```
CMD curl -f http://localhost:$HEALTHCHECK_SERVER_PORT/pulse || exit 1
```

**Comment by F4ever** [comment]
@vgorkavenko 
No pulse function just do `/pulse/` request to server that contains last pulse datetime.
All rest requests to this server just returns is last_puls_datetime expired or not

**Review by vgorkavenko — APPROVED** [review]

**Comment by vgorkavenko** [comment]
> @vgorkavenko No pulse function just do `/pulse/` request to server that contains last pulse datetime. All rest requests to this server just returns is last_puls_datetime expired or not

Yes. Thanks for the explanation!

File: src/modules/submodules/oracle_module.py [hunk 1]
```diff
@@ -110,6 +111,10 @@ def run_cycle(self, blockstamp: BlockStamp) -> ModuleExecuteDelay:
             logger.error({'msg': 'Keys API service returned incorrect number of keys.', 'error': str(error)})
         except ValueError as error:
             logger.error({'msg': 'Unexpected error.', 'error': str(error)})
+        else:
+            # Do pulse only if no exceptions only
```
Comment (skhomuti) [inline src/modules/submodules/oracle_module.py:115]: double "only" typo

**Review by skhomuti — APPROVED** [review]

File: src/modules/submodules/oracle_module.py (hunk 1 above)
Comment (F4ever) [reply src/modules/submodules/oracle_module.py]: fixed

**Pull request merged**